tests/clock/conftest.py

```diff
@@ -42,6 +42,13 @@ def restore_time(duthosts, ntp_server):
     """
     @summary: fixture to restore time after test (using ntp)
     """
+    logging.info('Check NTP server reachability')
+    duthost = duthosts[0]
```

Comment:
why only the first dut? the whole test is running only on the first one?

Reply:
These tests were designed to run all the DUT commands with ClockUtils.run_cmd(), so I've replaced direct duthost cmd run with run_cmd() method.